storm-core/src/clj/backtype/storm/converter.clj

```diff
@@ -0,0 +1,200 @@
+(ns backtype.storm.converter
```

Comment:
I personally would rather see the code use the Thrift objects directly.  One of the reasons I wanted to have Thrift over just using maps, was that it would force a developer to think about any changes that make.  To me having this here means that someone will add something to Assignment for example and not update this.  Then it will work just fine in local mode testing, but will not work in a real cluster, and would be a pain to debug.